src/doc/en/tutorial/afterword.rst

```diff
@@ -107,11 +107,9 @@ behaves differently from Python in several ways.
        10
 
 -  **Integer division:** The Python expression ``2/3`` does not
-   behave the way mathematicians might expect. In Python2, if ``m`` and
-   ``n`` are ints, then ``m/n`` is also an int, namely the quotient of ``m``
-   divided by ``n``. Therefore ``2/3=0``. In Python3, ``2/3`` returns the
-   floating point number ``0.6666...``. In both Python2 and Python3, ``//``
-   is the Euclidean division and ``2//3`` returns ``0``.
+   behave the way mathematicians might expect:  ``2/3`` returns the
+   floating point number ``0.6666...``. (Note that ``//``
+   is the Euclidean division and ``2//3`` returns ``0``).
```

Comment:
`returns ``0``.)` The period at the end should be inside the parentheses.

or just remove the parentheses.